RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerWeights.h

```diff
@@ -57,8 +57,9 @@ class EcalUncalibRecHitWorkerWeights : public EcalUncalibRecHitWorkerRunOneDigiB
                 EcalUncalibRecHitRecWeightsAlgo<EBDataFrame> uncalibMaker_barrel_;
                 EcalUncalibRecHitRecWeightsAlgo<EEDataFrame> uncalibMaker_endcap_;
 
- 		const EEShape testbeamEEShape; // used in the chi2
-                const EBShape testbeamEBShape; // can be replaced by simple shape arrays of floats in the future (kostas)
+                bool useDBShape;
+ 		EEShape testbeamEEShape; // used in the chi2
+                EBShape testbeamEBShape; // can be replaced by simple shape arrays of floats in the future (kostas)
```

Comment:
please use a consistent indentation

Reply:
the identation should be consistent, in the [raw file](https://raw.githubusercontent.com/theofil/cmssw/ecalrebase/RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerWeights.h)  it appears consistent

Reply:
you're mixing tabs and spaces, try to avoid this